### DISCUSSION TITLE
Improvements to Profile and Login Redirects

### DISCUSSION THREAD

**Discussion by jbranchaud** [eggheadio/egghead-next#1021] [General]
As of https://github.com/eggheadio/egghead-next/pull/1020, if you're already a pro user, you'll be redirected away from the pricing page.
- if you're signed in, you'll be taken to your profile page.
- if you're not signed in, you'll be taken to the login page.

How can we improve the experience and what is being communicated during those redirects?

### Here is what you see on the logged-in redirect

<img width="1206" alt="CleanShot 2021-12-27 at 10 53 16@2x" src="https://user-images.githubusercontent.com/694063/147495975-0f57c56c-bcf4-4773-919e-6481e4a05d47.png">

### Here is what you see on the logged-out redirect

<img width="1207" alt="CleanShot 2021-12-27 at 10 50 34@2x" src="https://user-images.githubusercontent.com/694063/147496013-a64eb670-3a01-4f8e-b3d4-ca5a56400009.png">

**Comment by joelhooks** [comment]
For me this is when “flash messages” ala Rails are really nice to give some feedback of what just happened


improving the profile page would make a very good project in general. 


bad 😵‍💫
